### ISSUE TITLE
Starfield artifacts with new 4952 driver

### ISSUE THREAD

**Issue by carloslomas0** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#579]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Starfield

### Game Platform [Required]

- [ ] Steam
- [X] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

Intel I7 10700K

### Graphic Card [Required]

Intel ARC A750

### GPU Driver Version [Required]

31.0.101.4952

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[Report.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13257455/Report.txt)

### Description and steps to reproduce [Required]

Getting artifact when playing starfield with 4952 driver:

![Screenshot 2023-11-04 012818](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/149896928/083c9aa4-1b91-4913-a94e-02d5b2c39802)

Also, can't broadcast or capture from intel arc control due to says HDR is active (My monitor doesn't support HDR so it can't be active) don't know if both issues could be related.

![Screenshot_2023-11-04_022308](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/149896928/e64deab4-2f9a-4ff6-9dac-f479f152e523)

Fixes that were tried:

Uninstalling and installing game.
Using DDU and installing 4900 driver, opening game, and then installing 4952 again to recompile shaders.

This fixes didn't worked only solution was to return to 4900 for now to fix artifacts in starfield. (Still can't broadcast or capture due to HDR error).

### Game graphic quality [Required]

- [X] Low
- [ ] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

Off

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

All settings in low.
Motion blur in low (not off).
VRS - On
Depth of field - On 
Upscaling - Off

**Comment by Karen-Intel** [comment]
Hey @carloslomas0 ty for your report!
FYI our team is already looking into this issue
Sharing internal tracking number FYI, 14020769808
As soon as I get any update I will let you know

Karen

**Comment by Nidzhun** [comment]
Not about this issue but i have performance regression in Starfield(about 15-20%) with 4952 driver on arc a770 le. I used ddu multiple times but seems that it is driver issue. I use windows 11 23h2.

**Comment by Karen-Intel** [comment]
@Nidzhun yeah that has been reported also. Our team is checking performance issues very actively
The internal report number for that issue is: 14020107573. You can find one or 2 threads here regarding that same issue

Karen

**Comment by carloslomas0** [comment]
Found the issue with both, artifacts in Starfield and HDR error message to capture or broadcast with ARC control:

![Screenshot 2023-11-06 230702](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/149896928/e2364c10-aee6-4a9a-827a-6ff4ae64db9c)

Seems this Windows feature is having issues with new 4952 driver, I turn it off and no artifacts  in Starfield and I can broadcast and capture with ARC Control now, feature works fine with 4900 driver.

**Comment by Karen-Intel** [comment]
@carloslomas0 ty! This is great information :) I will add it to our internal report as an update

Karen

**Comment by akarnokd** [comment]
FYI, 4953 WHQL is out supposedly fixing some color corruption issues.

**Comment by SavageWaffle** [comment]
This was fixed, and can be closed
